.github/workflows/build-dev.yml

```
@@ -100,19 +99,19 @@ jobs:
           prerelease: true
           body: ${{ env.VERSION }}
         env:
-          export PATH: $PATH:/usr/local/go/bin 
-              
+          export PATH: $PATH:/usr/local/go/bin
```

actionlint (1.7.12)

[error] environment variable name "export PATH" is invalid. '&', '=' and spaces should not be contained

(env-var)